recipe/meta.yaml

```diff
@@ -1,30 +1,30 @@
 {% set name = "dask-yarn" %}
-{% set version = "0.7.0" %}
+{% set version = "0.8.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: a40d05715616deac0f4a394e8a9dfd86f903bdf49cd1ce98c45a55e95a89ba49
+  sha256: 6c598f821cf0647f7a6c6155f43b5e79137b5adcc3dc51af7649003916688088
 
 build:
-  number: 1
+  number: 0
   skip: True  # [win or py<35]
   entry_points:
     - dask-yarn=dask_yarn.cli:main
-  script: python -m pip install --no-deps --ignore-installed .
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   host:
     - python
     - pip
   run:
     - python
-    - dask >=2.2.0
-    - distributed >=2.2.0
-    - skein >=0.5.0
+    - dask >=2.6.0
+    - distributed >=2.6.0
```

Comment:
Note that `dask` is a meta-package that includes `distributed`.   You may want `dask-core` here instead.